README.md

```diff
@@ -86,163 +85,175 @@ Some additionally important notes about how snap works:
+Type: collector
+Signed: false
+Loaded Time: Fri, 14 Oct 2016 10:54:07 PDT
 ```
 
-Let's look at what plugins you have loaded now:
+Verify plugins are loaded:
 ```
 $ snapctl plugin list
-NAME             VERSION         TYPE            SIGNED          STATUS          LOADED TIME
-mock             1               collector       false           loaded          Tue, 17 Nov 2015 14:08:17 PST
-passthru         1               processor       false           loaded          Tue, 17 Nov 2015 14:16:12 PST
-file             3               publisher       false           loaded          Tue, 17 Nov 2015 14:16:19 PST
+NAME      VERSION    TYPE         SIGNED     STATUS    LOADED TIME
+file      2          publisher    false      loaded    Fri, 14 Oct 2016 10:55:20 PDT
+psutil    8          collector    false      loaded    Fri, 14 Oct 2016 10:55:29 PDT
+```
+
+See which metrics are available:
+```
+$ snapctl metric list
+NAMESPACE                                VERSIONS
+/intel/psutil/cpu/cpu-total/guest        8
+/intel/psutil/cpu/cpu-total/guest_nice   8
+/intel/psutil/cpu/cpu-total/idle         8
+/intel/psutil/cpu/cpu-total/iowait       8
+/intel/psutil/cpu/cpu-total/irq          8
+/intel/psutil/cpu/cpu-total/nice         8
+/intel/psutil/cpu/cpu-total/softirq      8
+/intel/psutil/cpu/cpu-total/steal        8
+/intel/psutil/cpu/cpu-total/stolen       8
+/intel/psutil/cpu/cpu-total/system       8
+/intel/psutil/cpu/cpu-total/user         8
+/intel/psutil/load/load1                 8
+/intel/psutil/load/load15                8
+/intel/psutil/load/load5                 8
+...
 ```
 
-You now have one of each plugin type loaded into the framework. To begin collecting data, you need to create a task.
+NOTE: Plugin bundles are available for convenience in the Snap [GitHub release page](https://github.com/intelsdi-x/snap/releases), for the latest up to date version use the release/download in the [plugin catalog](#plugin-catalog).
 
 ### Running Tasks
-[Tasks](docs/TASKS.md) are most often shared as a Task Manifest and is written in JSON or YAML format.
-
-Create a task manifest file, for example `mock-file.yaml` with following content:
-```yaml
----
-  version: 1
-  schedule:
-    type: "simple"
-    interval: "1s"
-  max-failures: 10
-  workflow:
-    collect:
-      metrics:
-        /intel/mock/foo: {}
-        /intel/mock/bar: {}
-        /intel/mock/*/baz: {}
-      config:
-        /intel/mock:
-          name: "root"
-          password: "secret"
-      process:
-        -
-          plugin_name: "passthru"
-          config:
-            debug: true
-          process: null
-          publish:
-            -
-              plugin_name: "mock-file"
-              config:
-                file: "/tmp/snap_published_mock_file.log"
-                debug: true
-```
 
-and then start the task:
+Tasks can be created directly via `snapctl`, but due to length of task it makes more sense to load from a `Task Manifest` written in JSON or YAML format.
```

Comment:
"Tasks can be created directly via `snapctl`"

Adding the ability to generate a well-formed task manifest is a feature that should eventually be added but doesn't currently exist.